web/src/modules/common/components/Page.js

```diff
@@ -25,6 +25,7 @@ type PropsType = {|
   thumbnailSrcSet?: ?string,
   content: string,
   lastUpdate?: Moment,
+  showUpdateText?: boolean,
```

Comment:
I'd rename this to `showLastUpdateText` as that is what the component is called, but apart from that nicely done :+1:

Reply:
Makes sense, i updated it